pages/common/pulumi-new.md

```diff
@@ -0,0 +1,24 @@
+# pulumi new
+
+> Create a new Pulumi project.
+> More information: <https://www.pulumi.com/docs/iac/cli/commands/pulumi_new/>.
+
+- Choose a template interactively:
+
+`pulumi new`
+
+- Create a project from a specific template (e.g azure-python):
+
+`pulumi new {{provided-template}}`
+
+- Create a project from a local file:
+
+`pulumi new {{path/to/templates/aws-typescript}}`
+
+- Create a project from a specific source control location:
+
+`pulumi new {{https://github.com/user/repo}}`
```

Comment:
Instructions unclear. Does it accept only github links, or any Git repo, or can I use SVN too?

```suggestion
- Create a project from a Git repository:

`pulumi new {{url}}`
```

Reply:
Good point. Any git repo will do, private or hosted on some public site.